phnt/include/ntldr.h

```diff
@@ -662,12 +662,12 @@ typedef struct _PS_SYSTEM_DLL_INIT_BLOCK
             ULONG Reserved : 31;
         };
     };
-    PS_MITIGATION_OPTIONS_MAP MitigationOptionsMap;
-    ULONG_PTR CfgBitMap;
+    PS_MITIGATION_OPTIONS_MAP MitigationOptionsMap; // current size since 20H1
+    ULONG_PTR CfgBitMap; // current offset since 20H1
     ULONG_PTR CfgBitMapSize;
     ULONG_PTR Wow64CfgBitMap;
     ULONG_PTR Wow64CfgBitMapSize;
-    PS_MITIGATION_AUDIT_OPTIONS_MAP MitigationAuditOptionsMap; // REDSTONE3
+    PS_MITIGATION_AUDIT_OPTIONS_MAP MitigationAuditOptionsMap;
```

Comment:
Why remove the version?

Reply:
Because (at its current offset) the field is usable only since the same version (20H1) as a marked a few fields above. The full comment would be `since REDSTONE3, current offset since 20H1`